### PR TITLE
Patch the readme per the templates

### PR DESCRIPTION
**Pull request by nikkictl**
Signed-off-by: Nikki Attea <nikki@sensu.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,15 +43,15 @@ Flags:
 
 ### Asset registration
 
-Assets are the best way to make use of this plugin. If you're not using an asset, please consider
-doing so! If you're using sensuctl 5.13 with Sensu Backend 5.13 or later, you can use the following
-command to add the asset:
+[Sensu Assets][10] are the best way to make use of this plugin. If you're not using an asset, please
```
Comment (ccressent) [inline README.md]: `10` is a link to the Wavefront homepage

**Review by ccressent — APPROVED** [review]
Looks good, just need to update the "Sensu Assets" link!

File: README.md (hunk 1 above)
Comment (nikkictl) [inline README.md]:
```suggestion
[Sensu Assets][14] are the best way to make use of this plugin. If you're not using an asset, please
```

File: README.md [hunk 2]
```diff
@@ -96,4 +96,5 @@ For more information about contributing to this plugin, see [Contributing][1].
 [9]: https://docs.sensu.io/sensu-go/latest/reference/handlers/#how-do-sensu-handlers-work
 [10]: https://www.wavefront.com/
 [11]: https://github.com/sensu/sensu-go
-[12]: https://docs.wavefront.com/proxies.html
\ No newline at end of file
+[12]: https://docs.wavefront.com/proxies.html
+[13]: https://bonsai.sensu.io/assets/sensu/sensu-wavefront-handler
```
Comment (nikkictl) [inline README.md]:
```suggestion
[13]: https://bonsai.sensu.io/assets/sensu/sensu-wavefront-handler
[14]: https://docs.sensu.io/sensu-go/latest/reference/assets/
```

Comment (nikkictl) [inline README.md:100]:
```suggestion
[13]: https://bonsai.sensu.io/assets/sensu/sensu-wavefront-handler
[14]: https://docs.sensu.io/sensu-go/latest/reference/assets/
```

**Pull request merged**